src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/pyproject.toml

```diff
@@ -53,7 +52,7 @@ dash-mantine-components = "^0.12.1"
 dash-uploader = "^0.6"
 {% elif cookiecutter.dash_ui == "awc" %}
 ansys-web-components-dash = {version = ">=0.0.dev4", source = "solutions-private-pypi"}
-ansys-solutions-dash-super-components = {version = "^0.1.dev0", python = ">=3.9,<3.11"}
+ansys-solutions-dash-super-components = {version = "0.1.dev0", source = "solutions-private-pypi", python = ">=3.10,<3.11"}
```

Comment:
this is a duplicate. I guess we should remove this!

Reply:
This is not a duplicate but the config for the AWC mode.